pyproject.toml

```diff
@@ -13,34 +13,18 @@ build-backend = "setuptools.build_meta"
 line-length = 79
 target-version = ['py37', 'py38']
 include = '\.pyi?$'
-exclude = '''
```

Comment:
Unfortunately, whereas the pre-commit part makes sense to me,
I just don't feel anyway competent to judge on this bit.

Could we have avoided changes to pyproject.toml + done those separately (cos then I might merge) ?

Reply:
This part is actually redundant.

Note the change in directive from `exclude` to `extend-exclude`. We're now getting a lot for free, whereas before we were being unnecessarily explicit.